### PR TITLE
Update instructors and attribution

### LINKED ISSUE #43: Update README.md
- [x] include new trainers and reference to new training in go-slides README.md
- [x] update Pre-PR checklist go-training. add: review other PR section
- [x] update Build, test etc. section: remove `install` and add `build`

### LINKED ISSUE #55: Add/remove instructors
Add Josh and Dan
Remove Angelo

### PR DESCRIPTION
**Pull request by juliaogris**
### Changes proposed in this PR

- Update instructor names on title and contact slide
- Update attribution on README.md

View at: https://pr56-dot-anzedu.appspot.com

Fixes #55, fixes #43

### CODE REVIEW

File: content/gocourse-2019-05-melbourne.slide [hunk 1]
```diff
@@ -61,7 +62,8 @@ The last 30 minutes of each talk are reserved for Q&A and help.
 Please join [[https://anzengineering.slack.com/messages/CFCPLHFT2/convo/CFCPLHFT2-1557100844.007600/][#go-course]] on [[https://anzengineering.slack.com]]
 
 *Instructors*
-[[https://anzengineering.slack.com/team/UEVMPCL2C][@Angelo]], angelo.perera@anz.com
+[[https://anzengineering.slack.com/team/UCUUKEWGZ][@Dan]], Daniel.Cantos@anz.com
+[[https://anzengineering.slack.com/team/UJC0EUZ3P][[@Joshua Carpeggiani]], Joshua.Carpeggiani@anz.com
```
Comment (camscale) [inline content/gocourse-2019-05-melbourne.slide]: This has one too many left-square-brackets before "@ Joshua"

**Review by camscale — COMMENTED** [review]
Typo in commit message: s/Attirbution/Attribution/

**Review by camscale — APPROVED** [review]
LGTM

**Comment by juliaogris** [comment]
For you sir, @camh-anz, I'm checking out merge commits....

**Pull request merged**